<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwAddEditBatchDialogUi</class>
 <widget class="QDialog" name="AwAddEditBatchDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Process input</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelDir">
            <property name="text">
             <string>Directory:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editDir"/>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBrowseDir">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayoutFiles">
          <item row="2" column="0">
           <widget class="QPushButton" name="buttonAddFromDir">
            <property name="toolTip">
             <string>Click to pick a root directory</string>
            </property>
            <property name="text">
             <string>Add From Directory</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="buttonAddFiles">
            <property name="toolTip">
             <string>Click to add files</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButton">
            <property name="toolTip">
             <string>Remove selected filles</string>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelFiles">
            <property name="text">
             <string>Files:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QTableWidget" name="tableWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <column/>
            <column/>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupParameters">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resources/AnyWave.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>AwAddEditBatchDialogUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>AwAddEditBatchDialogUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>705</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddFromDir</sender>
   <signal>clicked()</signal>
   <receiver>AwAddEditBatchDialogUi</receiver>
   <slot>addFilesFromDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddFiles</sender>
   <signal>clicked()</signal>
   <receiver>AwAddEditBatchDialogUi</receiver>
   <slot>addFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AwAddEditBatchDialogUi</receiver>
   <slot>removeFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFilesFromDir()</slot>
  <slot>addFiles()</slot>
  <slot>removeFiles()</slot>
 </slots>
</ui>
